### ISSUE TITLE
Lost styling on Project page

### ISSUE THREAD

**Issue by skleinfeldt** [jazkarta/jazkarta.com#125]
Just under title, Project Type, Client, Location, and Technology have become white and the wrong font. For example https://jazkarta.com/projects/jazkarta-website

**Comment by cguardia** [comment]
Which font should it be?

**Comment by skleinfeldt** [comment]
Dunno, but the attached design comp will give you the idea (even though it's old and has that orange.)
[Jazkarta Site v14.pdf](https://github.com/jazkarta/jazkarta.com/files/507798/Jazkarta.Site.v14.pdf)

**Comment by cguardia** [comment]
Seems related to plone.app.imagecropping. Did we make any changes that used that recently?

**Comment by skleinfeldt** [comment]
Yes! I was wondering because that's the only thing I can think of that's changed. @alecpm can you weigh in here?

**Comment by skleinfeldt** [comment]
Fairly recently, @alecpm did a version upgrade - right Alec?

It would (obviously) be nice to have this fixed by the conference.

**Comment by alecpm** [comment]
For now I've enabled the image cropping resource bundle only when logged in, which appears to fix the issue for now. It looks like the p.a.imagecropping css includes some bootstrap css (labels.less) that turns everything with class "label" white.

Appears to be a known issue:

https://github.com/collective/plone.app.imagecropping/issues/78

**Comment by skleinfeldt** [comment]
thanks @alecpm

**Comment by skleinfeldt** [comment]
It looks like this issue was fixed last November:  collective/plone.app.imagecropping#78

Reopening this with a request to upgrade to the fixed version. I think this is dependent on upgrading our site to Plone 5.1.

**Comment by davisagli** [comment]
Seems to be working now that we upgraded.

**Comment by skleinfeldt** [comment]
But we haven't upgraded the live site yet, keeping this open so I remember why.
